src/services/EdFi.DataManagementService.Api/Configuration/RateLimitOptions.cs

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+namespace EdFi.DataManagementService.Api.Configuration
+{
+    public class RateLimitOptions
+    {
+        public const string RateLimit = "RateLimit";
+
+        //Number of requests per window
+        public int PermitLimit { get; set; }
```

Comment:
When I try setting a negative value, I get an unhandled exception and the application doesn't startup. Failing to start makes sense, and the unhandled exception should be covered in the upcoming DMS-8 work. No concern in this ticket. @CSR2017 in the DMS-8 work, you can use this test case when validating the logging and exception handling.